docs/heritage.rst

```diff
@@ -0,0 +1,33 @@
+*************
+Code Heritage
+*************
+
+``pfsspy``
+==========
+
+As of initial release, the main component of ``sunkit-magex`` (`sunkit_magex.pfss`) is forked from `pfsspy` package (archived as of August 24, 2023).
+`pfsspy` implemented the Potential Field Source Surface (PFSS) model, a widely used method to extrapolate the magnetic fields of the solar corona.
+`pfsspy` was developed, integrated with `sunpy` and maintained by `David Stansby <https://www.davidstansby.com/>`_, based on an `original PFSS implementation <https://doi.org/10.5281/zenodo.1472183>`_  by `Anthony Yeates <https://www.maths.dur.ac.uk/users/anthony.yeates/>`_.
```

Comment:
```suggestion
`pfsspy` was developed, integrated with `sunpy` and maintained by `David Stansby <https://www.davidstansby.com/>`__, based on an `original PFSS implementation <https://doi.org/10.5281/zenodo.1472183>`__  by `Anthony Yeates <https://www.maths.dur.ac.uk/users/anthony.yeates/>`__.
```
So this is something that sphinx recommends doing to urls that are only used one.